src/analysis/useless_initializer.d

```diff
@@ -5,13 +5,20 @@
 module analysis.useless_initializer;
 
 import analysis.base;
+import containers.dynamicarray;
+import containers.hashmap;
 import dparse.ast;
 import dparse.lexer;
+import std.algorithm : among, canFind;
+import std.algorithm.iteration : filter;
```

Comment:
Why not simply importing entire `std.algorithm` - it's very commonly used... and saves the pain of updating this list all the time.